node/sc/vt_recovery.go

```diff
@@ -46,8 +46,8 @@ type valueTransferRecovery struct {
 
 	child2parentHint *valueTransferHint
 	parent2childHint *valueTransferHint
-	childEvents      []*RequestValueTransferEvent
-	parentEvents     []*RequestValueTransferEvent
+	childEvents      []interface{}
+	parentEvents     []interface{}
```

Comment:
Why did you change this VTRecovery struct?

Reply:
Now that the number of the listening events is not the only one, I changed it to an interfaced structure.